support-frontend/app/controllers/Application.scala

```diff
@@ -234,6 +239,45 @@ class Application(
     )()).withSettingsSurrogateKey
   }
 
+  def contributionsCheckoutInEpic(countryCode: String): Action[AnyContent] = PrivateAction { implicit request =>
```

Comment:
This is currently a `PrivateAction` because we were running into issues with the `SetCookiesCheck` filter. However given the potential number of views this will get, perhaps we want to figure out a better solution here?

Reply:
(ie we want it to be cached?)